### PR TITLE
Limit number of hops in fuzzing

### PR DESCRIPTION
**Pull request by e00E**
We saw this from the fuzzer pod running in kubernetes.

### Test Plan
Run fuzzer.

### CODE REVIEW

**Review by nlordell — APPROVED** [review]

File: pricegraph/fuzz/fuzz_targets/pricegraph.rs [hunk 1]
```diff
@@ -59,6 +59,11 @@ fuzz_target!(|arguments: Arguments| {
                     return;
                 }
             }
+            if let Some(hops) = hops {
+                if hops > 30 {
+                    return;
+                }
+            }
             pricegraph.transitive_orderbook(market, hops, spread);
```
Comment (nlordell) [inline pricegraph/fuzz/fuzz_targets/pricegraph.rs:67]: Could you just
```suggestion
            pricegraph.transitive_orderbook(market, hops % 30, spread);
```

Not sure what works better with the fuzzer...

**Comment by nlordell** [comment]
Also, 0 hops might not make sense either.

**Review by fleupold — APPROVED** [review]

File: pricegraph/fuzz/fuzz_targets/pricegraph.rs (hunk 1 above)
Comment (e00E) [reply pricegraph/fuzz/fuzz_targets/pricegraph.rs:67]: Dunno which works better but yeah let's use that suggestion.

Comment (nlordell) [reply pricegraph/fuzz/fuzz_targets/pricegraph.rs]: Sorry @e00E, I removed the `merge when green` label as I think there is an issue. It looks like the currently:
- We are using the `if` statement and the modulo expression
- We are computing a modulo on an `Option<usize>`, my suggestion was more pseudo-code than real code :see_no_evil:

Comment (e00E) [reply pricegraph/fuzz/fuzz_targets/pricegraph.rs]: Oops, thanks. Fixed it.

**Pull request merged**